oap-server/server-starter/src/main/resources/otel-oc-rules/k8s-instance.yaml

```diff
@@ -17,6 +17,7 @@ filter: "{ tags -> tags.job_name in [ 'kubernetes-cadvisor', 'kube-state-metrics
 expSuffix: |-
   service(['cluster' , 'service'], '::', Layer.K8S_SERVICE)
   .instance(['cluster', 'service'], '::', ['pod'], '', Layer.K8S_SERVICE, { tags -> ['pod': tags.pod, 'namespace': tags.namespace] })
+  .drop()
```

Comment:
`drop` is making this strange. 2 perspective
1. User would ask why we generate a metric and then drop
2. Meter system engine is still running for this metric. AKA, this `drop` only means not aggregation.

How about we keep this metric for now?